content/medo_manual/werken_met_deelnemers.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Werken met deelnemers
+sort: 300
+---
+
+* TOC
+{:toc}
+
+## Deelnemerslijst
+
+Op deze pagina bevindt zich een tabel met de deelnemers van het project. Door
+op de tabelkoppen te klikken kan de tabel gesorteerd worden. Daarnaast is het
+mogelijk om door de tabel te zoeken, waarbij door data van meerdere kolommen
+gezocht kan worden. Verzamelaars en coördinatoren kunnen de voornaam,
+achternaam en het e-mailadres van deelnemers bekijken.
+
+<img src='/assets/images/screenshots/medo/deelnemerslijst.png' />
+
+### Privacy
+
+Door de optie *Persoonsgegevens tonen* uit te vinken kunnen deze gegevens ook verborgen
+worden. Door op een rij te klikken gaat u naar de detailpagina van die deelnemer.
+
+### Statistieken
+
+Rechts in de tabel zijn drie statistieken opgenomen:
+
+* **Gemiste metingen** geeft aan hoeveel metingen er gemist zijn.
+* **Ingevulde metingen** geeft aan hoeveel er ingevuld zijn.
+* **Totaal metingen** geeft aan hoeveel metingen er in totaal klaargezet zijn.
+
+### Import en Export
+
+Ook is het voor verzamelaars en coördinatoren mogelijk om de deelnemerslijst te
+exporteren en nieuwe deelnemers te importeren. Als u op de **Lijst importeren**
+knop drukt krijgt u een pagina te zien met toelichting over het aan te leveren
+csv-bestand. Door op de **Lijst exporteren** knop te drukken krijgt u een
+csv-bestand met daarin alle gegevens van de deelnemers in hetzelfde formaat als
+het csv-bestand voor importeren.
+
+<div class="alert alert-error">
+  <strong>Let op!</strong>
+  Het is van belang dat u het csv-bestand van de export niet in hetzelfde
+  project probeert te importeren. Daardoor zal ongewenste dubbele data ontstaan.
+</div>
+
+## Deelnemer toevoegen
```

Comment:
Ah, daar heb je gelijk in. Ik zag niet dat persoonsgegevens hier een subkopje was.